src/components/page-intro.tsx

```diff
@@ -12,9 +11,39 @@ import '../styles/components/page-intro.scss';
 
 const chevronSize = 20;
 
-const PageIntro = ({ title, resultsCount, children, links, showContent }) => {
-  const [displayContent, setDisplayContent] = useState(showContent);
+type PageIntroProps = {
+  /**
+   * The title, works as a trigger to open/close
+   */
+  title: ReactNode;
+  titlePostscript?: ReactNode;
+  /**
+   * Number of results
+   */
+  resultsCount?: number;
+  /**
+   * Content revealed on toggle
+   */
+  children?: ReactNode;
+  /**
+   * Links revealed on toggle
+   */
+  links?: { title: string; destination: string }[];
+  /**
+   * To show the content by default or not
+   */
+  showContent?: boolean;
+};
 
+const PageIntro: FC<PageIntroProps> = ({
+  title,
+  resultsCount = 0,
+  children,
+  links,
+  showContent = false,
+  titlePostscript,
+}) => {
+  const [displayContent, setDisplayContent] = useState(showContent);
   return (
     <div className="page-intro">
       <h2>
```

Comment:
This should default to `h1`, and be configurable through props (see `headingLevel` prop in `HeroContainer` or `Tile` for example) but should have the `.big` classname to define its size regardless of the heading level

Reply:
Good idea - updated

Reply:
forgot to do the `.big` aspect. doing now

Reply:
Oh, right, forgot about that in my review now